Header header

float64[] timestamps
geometry_msgs/Vector3[] angular_velocities
geometry_msgs/Vector3[] linear_accelerations
